assets/app/scripts/extensions/javaLink.js

```diff
@@ -40,25 +40,27 @@ angular.module('openshiftConsoleExtensions', ['openshiftConsole'])
       if (!jolokiaPort) {
         return;
       }
-      var pod = $scope.$eval('pod');
+      var pod = $scope.$eval('podTemplate');
       if (!pod) {
         return;
       }
       var podId = pod.metadata.name;
       var namespace = pod.metadata.namespace;
-      $scope.jolokiaUrl = new URI(ProxyPod(namespace, podId, jolokiaPort.containerPort)).segment('jolokia/').toString();
+      $scope.jolokiaUrl = ProxyPod(namespace, podId, jolokiaPort.containerPort).segment('jolokia/').toString();
       $scope.gotoContainerView = function($event, container, jolokiaUrl) {
         $event.preventDefault();
         $event.stopPropagation();
         var returnTo = window.location.href;
         var title = container.name || 'Untitled Container';
+        var token = LocalStorageUserStore.getToken();
         var targetURI = new URI().path(BaseHref)
                                  .segment('java')
                                  .segment('index.html')
                                  .query({
                                    jolokiaUrl: jolokiaUrl,
                                    title: title,
-                                   returnTo: returnTo
+                                   returnTo: returnTo,
+                                   token: token
```

Comment:
I don't want the access token in the URL (both for display purposes, and in the query, it could get sent to the server and logged)... is there any other way to get it?